code/obj/storage/wardrobes.dm

```diff
@@ -126,7 +126,6 @@
 	/obj/item/clothing/head/bio_hood/janitor = 1,
 	/obj/item/clothing/mask/gas = 1,
 	/obj/item/clothing/gloves/long = 1,
-	/obj/item/clothing/shoes/galoshes = 1,
```

Comment:
This closet spawns in a z5 prefab, and is uncommon/interesting enough to deserve keeping the galoshes in it, I think.